jax/numpy/lax_numpy.py

```diff
@@ -2034,25 +2034,25 @@ def append(arr, values, axis=None):
 
 
 @_wraps(onp.dot)
-def dot(a, b):  # pylint: disable=missing-docstring
+def dot(a, b, precision=None):  # pylint: disable=missing-docstring
```

Comment:
How will this show in the documentation? numpy.dot does not have a precision argument, right? Then maybe the documentation should somehow reference the documentation of lax.dot, which does have a precision.

Reply:
I suspect the inevitable thing for us to do is to start writing our own doc strings for these functions, rather than reusing NumPy's doc strings via the `@_wraps` decorator. There are quite a few places where the NumPy doc strings are misleading with respect to JAX.

Reply:
I added a short sentence to every docstring referencing `jax.lax.dot` (with a link that should work in Sphinx), but I have to say, the documentation about precision even in `lax.dot` is pretty lacking. It would be nice for somebody who knows exactly how it works to add a few details.